packages/data-point/lib/resolve/reducer-path.js

```diff
@@ -1,56 +1,16 @@
 'use strict'
 
-const _ = require('lodash')
 const utils = require('../utils')
 
 /**
- * resolve the JSON Object path from given value
- *
- * @param {Object} value
- * @param {string} jsonPath
- * @returns
- */
-function resolveObjectPath (acc, jsonPath, reducerPath) {
-  if (typeof reducerPath === 'undefined') {
-    reducerPath = {}
-  }
-
-  const type = reducerPath.type || ''
-  const asCollection = reducerPath.asCollection || false
-
-  if (jsonPath === '.' || _.isEmpty(jsonPath)) {
-    return acc.value
-  }
-
-  if (jsonPath.substring(0, 2) === '..') {
-    return _.get(acc, jsonPath.slice(2))
-  }
-
-  if (type === 'ReducerPath' && asCollection) {
-    if (!Array.isArray(acc.value)) {
-      return null
-    }
-
-    return _.map(acc.value, jsonPath)
-  }
-
-  return _.get(acc.value, jsonPath)
-}
-
-module.exports.resolveObjectPath = resolveObjectPath
-
-/**
- * resolve a reducer json path from a accumulator object
- *
- * @param {Object} store
- * @param {function} resolveReducer
- * @param {Accumulator} accumulator current accumulator
+ * @param {Accumulator} accumulator - current accumulator
  * @param {ReducerPath} reducerPath
  * @returns {Promise<Accumulator>}
  */
-function resolve (store, resolveReducer, accumulator, reducerPath) {
-  const value = resolveObjectPath(accumulator, reducerPath.name, reducerPath)
-  return Promise.resolve(utils.set(accumulator, 'value', value))
+function resolve (accumulator, reducerPath) {
```

Comment:
Excuse my ignorance here, but will this be a breaking change or does the change introduced in this PR in `~/packages/data-point/lib/resolve/reducer.js` capture the entire use of this method?

Reply:
It's not a breaking change because this function is only used internally